### PR TITLE
feat: replaced notification component  with UI library's

### LINKED ISSUE #394: [OSD] Feature Request: Replace current Notifications component with UI-library's Notifications component - DatePickerPoint
### Is your feature request related to a problem? Please describe.

_No response_

### Describe the solution you'd like.

_No response_

### Describe alternatives you've considered

_No response_

### Additional context.

_No response_

### PR DESCRIPTION
**Pull request by thisiskaransgit**
Signed-off-by: Karan Singh Thakur <karanthakur.work@gmail.com>

## Description

This PR fixes #394

Replaced all of the Notification component instances in DatePickerPoint.vue  with UI-library's Notifications component

## Additional Context (Please include any Screenshots/gifs if relevant)

Screenshot from 2022-12-03 19-27-31

### CODE REVIEW

**Comment by chicken-biryani** [comment]
@Crystalsage PTAL?

**Review by Crystalsage — APPROVED** [review]
LGTM. Thanks for working on this @thisiskaransgit.

@chicken-biryani

File: flint.ui/src/components/Datepicker/DatepickerPoint.vue [hunk 1]
```diff
@@ -63,9 +63,11 @@ export default {
         selectedStartDate.value = val
         console.log(selectedStartDate.value)
         if (date_diff.value < 0) {
-          notification.error({
+          useToast({
+            type: 'error',
+            title: 'Error',
```
Comment (Janvi-Thakkar) [inline flint.ui/src/components/Datepicker/DatepickerPoint.vue:68]: Hello @thisiskaransgit, Thanks for working on it! I would suggest to change the title from "Error" to "Invalid Start Date".

File: flint.ui/src/components/Datepicker/DatepickerPoint.vue [hunk 2]
```diff
@@ -79,9 +81,11 @@ export default {
         selectedEndDate.value = val
         console.log(selectedEndDate.value)
         if (date_diff.value < 0) {
-          notification.error({
+          useToast({
+            type: 'error',
+            title: 'Error',
```
Comment (Janvi-Thakkar) [inline flint.ui/src/components/Datepicker/DatepickerPoint.vue:86]: Here we can change from "Error" to "Invalid End Date".

File: flint.ui/src/components/Datepicker/DatepickerPoint.vue (hunk 1 above)
Comment (thisiskaransgit) [reply flint.ui/src/components/Datepicker/DatepickerPoint.vue]: Okay

File: flint.ui/src/components/Datepicker/DatepickerPoint.vue (hunk 2 above)
Comment (thisiskaransgit) [reply flint.ui/src/components/Datepicker/DatepickerPoint.vue]: Done

**Comment by Janvi-Thakkar** [comment]
Hello @thisiskaransgit, Thanks for working on this. LGTM:)

**Review by Janvi-Thakkar — APPROVED** [review]
Hello @thisiskaransgit, Thanks for working on this. LGTM:)

**Comment by Janvi-Thakkar** [comment]
@thisiskaransgit, Can you please resolve the conflicts so that we can merge it?

**Comment by thisiskaransgit** [comment]
Hey @Janvi-Thakkar, I had to revert the changes to resolve the conflicts, I've made a fresh PR, please take a look at  #443,

**Pull request closed without merging**